### ISSUE TITLE
Are you still active ? Important !

### ISSUE THREAD

**Issue by Ahmed-Dzair-Dev** [suntorvic/nugem#3]
I have forked your engine from this guy @humbertodias " sdl-nugem " i did alot of changes on your code .

I had to mention that i sucesed to run the engine in linux but it still have somebreak points causing a Segment Fault .

![Screenshot_20241225_151017.jpg](https://github.com/user-attachments/assets/9ca21a10-da02-4faf-a5b5-409f1ad747df)

Here is a video : [nugem engine test on linux](https://youtu.be/w5hy2jv6h8w?si=Zy9iNsfAVrHUkrKk)

**Comment by Ahmed-Dzair-Dev** [comment]
@flmmkch

**Comment by flmmkch** [comment]
Hi @Ahmed-Dzair-Dev thanks for reporting this issue. However I do not maintain this project anymore, I went further and better with the Rust project on https://github.com/flmmkch/nugem-rs

**Comment by Ahmed-Dzair-Dev** [comment]
I see .

**Comment by Ahmed-Dzair-Dev** [comment]
Any simple tutorial to compile this project ? Because i never tryed this programing language before. @flmmkch

**Comment by flmmkch** [comment]
> Any simple tutorial to compile this project ? Because i never tryed this programing language before. @flmmkch

Yes, the official Rust documentation is very good: https://doc.rust-lang.org/cargo/getting-started/index.html

**Comment by Ahmed-Dzair-Dev** [comment]
libEGL warning: DRI2: failed to authenticate
[2025-02-06T09:40:00Z ERROR nugem::game::scene::loading] Error loading next scene: No characters found in data directories.

**Comment by flmmkch** [comment]
@Ahmed-Dzair-Dev I don't know about the DRI2 warning but you would need to pass --data myDir/ to the command, and then add characters in subdirectories of myDir/
